files/var/www/ks/bootstrap_simp_client

```diff
@@ -465,8 +465,15 @@ EOM
   def set_system_time
     return if @options[:ntp_servers].nil? or @options[:ntp_servers].empty?
     info(title("Setting the system time against #{@options[:ntp_servers].join(' ')}", 2))
-    # this can fail if ntpd is up and running, so only log any failures
-    result = execute("/usr/sbin/ntpdate -b #{@options[:ntp_servers].join(' ')}")
+    # this can fail if the time service is up and running, so only log any failures
```

Comment:
templates/www/ks/runpuppet.erb was not similarly adjusted.  Do we want to remove that deprecated file at this point?  We don't recommend using it, because it does not work well on Systemd...